frontend/pages/index.tsx

```diff
@@ -30,21 +31,28 @@ const Home: NextPage = () => {
             FiberDiameter!
           </p>
         </h1>
-        <h2>
-          A deep learning-assisted app to{" "}
-          <span className='underline decoration-2 decoration-white-500'>
-            measure
-          </span>{" "}
-          your{" "}
-          <span className='underline decoration-2 decoration-white-500'>
-            fibers
-          </span>
+        <h2 className="h2-center">
+          Measure the diameter of fibers in microscopy images<br/>
```

Comment:
electron microscopy != microscopy 
Maybe better to put "Measure the diameter of fibers in electron microscopy images"